package.json

```
@@ -22,24 +23,81 @@
     "analyze": "./script/webpack-analyze"
   },
   "dependencies": {
+    "@apollo/client": "^3.3.7",
     "@module-federation/utilities": "^1.7.0",
+    "@novnc/novnc": "1.3",
```

```suggestion
    "@novnc/novnc": "1.3.0",
```

While for NPM those two are identical, RPM is stricter, and our tooling currently does not translate it well.